notebooks/official/matching_engine/sdk_matching_engine_create_stack_overflow_embeddings.ipynb

```diff
@@ -0,0 +1,1131 @@
+{
```

Comment:
<p>Nit, add in the article in bold "These are uploaded to <strong>the</strong> Vertex AI Matching Engine service"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1575/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done.</p>